client/test/e2e/depots_management/depots_management.spec.js

```diff
@@ -0,0 +1,49 @@
+/* jshint expr: true */
+/* describe, it, element, by, beforeEach, inject, browser */
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+var uuid = require('node-uuid');
+
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+var FormUtils = require('../shared/FormUtils');
+var components = require('../shared/components');
+
+describe.only('Depots management tests suit :: ', function () {
+
+  var PATH = '#/depots_management';
+
+  var depot = {
+    text : 'E2E_new_depot',
+    is_warehouse : 0
+  };
+
+  var updateDepot = {
+    text : ' => E2E_updated_depot',
+    is_warehouse : 1
+  };
+
+  /** navigate to the depots management module */
+  beforeEach(function () {
+    browser.get(PATH);
+  });
+
+  it('successfully creates a new depot', function () {
+    FormUtils.buttons.create();
+    FormUtils.input('DepotCtrl.depot.text', depot.text);
+    FormUtils.input('DepotCtrl.depot.is_warehouse', depot.is_warehouse);
```

Comment:
Does this actually work?  I think you might need to click the checkbox, not send '0' or '1' to it.  If this does work, please send a comment telling me it does.  Otherwise, try an `element().click()` method.